README.md

```diff
@@ -1,18 +1,110 @@
+
+### Quantum Developer Internships
+
+Developer interns will have the opportunity to focus on kernel, algorithm, or model development. Areas of focus could include quantum machine learning, quantum biology and health informatics, quantum algorithms for chemistry, quantum simulations, quantum optimization, cloud microservices, and cloud API (among other areas).
+
+Locations:
+
+- IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
+- IBM Research — Almaden (San Jose, California, USA)
+
+PhD Quantum Developer Summer Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/962210/PhD-Quantum-Developer-Summer-Intern-2021/
+
+Graduate Quantum Developer Summer Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/962202/Graduate-Quantum-Developer-Summer-Intern-2021/
+
+Undergraduate Quantum Developer Summer Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/962204/Undergraduate-Quantum-Developer-Summer-Intern-2021/
+
+### Quantum Engineer Internships
+
+Engineering interns will have the opportunity to focus on hardware of software engineering. Areas of engineering could include superconducting qubits, quantum software architecture, cryoelectronic circuit design, quantum microwave engineering, and quantum FPGA engineering (among other areas).
+
+Locations:
+
+- IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
+- IBM Research — Almaden (San Jose, California, USA)
+- IBM Systems Lab — Rochester, Minnesota, USA
+- IBM Systems Lab — Poughkeepsie, New York, USA
+
+PhD Quantum Engineering (United States):  
+https://careers.ibm.com/ShowJob/Id/962213/PhD-Quantum-Engineering-Summer-Intern-2021/
+
+Graduate Quantum Engineering Summer Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/962214/Graduate-Quantum-Engineering-Summer-Intern-2021/
+
+Undergraduate Quantum Engineering Summer Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/962215/Undergraduate-Quantum-Engineering-Summer-Intern-2021/
+
+### Quantum Designer Internships
+
+Locations:
+
+- IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
+- IBM Research — Austin, Texas, USA
+
+IBM Quantum Design Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/982781/IBM-Quantum-Design-Intern-2021/
+
+### Quantum Offering Manager Internships
+
+Locations:
+
+- IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
+- IBM Research — Almaden (San Jose, California, USA)
+
+IBM Quantum Offering Manager Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/981532/IBM-Quantum-Offering-Manager-Intern-2021/
+
+### Quantum Community Advocate Internships
+
+Locations:
+
+- IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
+- IBM Research — Almaden (San Jose, California, USA)
+
+IBM Quantum Community Advocate Intern (United States):  
+https://careers.ibm.com/ShowJob/Id/980793/IBM%20Quantum%20Community%20Advocate%20Intern%20%20%202021
+
+## Early Professional Roles
+
```

Comment:
We have Quantum Specific Bulk Requisites that may be better to include than the wider research reqs. We don't have a separate postdoc one, but we match up fairly well with the others, and have a developer role also.

IBM Quantum Research Staff Member - 2021
https://careers.ibm.com/ShowJob/Id/986522/IBM-Quantum-Research-Staff-Member-2021/

IBM Quantum Entry Level Hardware Developer - 2021
https://careers.ibm.com/ShowJob/Id/986523/IBM-Quantum-Entry-Level-Hardware-Developer-2021/

IBM Quantum Entry Level Developer - 2021
https://careers.ibm.com/ShowJob/Id/981311/IBM-Quantum-Entry-Level-Developer-2021/

IBM Quantum Entry Level Engineer - 2021
https://careers.ibm.com/ShowJob/Id/980973/IBM-Quantum-Entry-Level-Engineer-2021/